src/client.js

```diff
@@ -158,12 +161,15 @@ export class Saturn {
    * @returns {Promise<object>}
    */
   async fetchCID (cidPath, opts = {}) {
-    const [cid] = (cidPath ?? '').split('/')
-    CID.parse(cid)
-
-    const jwt = await getJWT(this.opts, this.storage)
+    const options = Object.assign({}, this.opts, opts)
+    if (!opts.originFallback) {
+      const [cid] = (cidPath ?? '').split('/')
+      CID.parse(cid)
+      const jwt = await getJWT(this.opts, this.storage)
+      options.jwt = jwt
+    }
+    options.format = opts.format ? opts.format : 'car'
```

Comment:
This will still return 'car' when we pass { format: null }. Now i'm wondering if it's better to create a client side only 'flat' format to avoid falsy checks defaulting to 'car', when in fact we want flat files.

Reply:
Addressed this. 

I think adding explicit file formats is better to identify `flat` or `car`. We only currently support car or flat though so I think the condition here is sufficient and we can change this later if needed if we start supporting more file formats.

Reply:
we'll eventually need to support `raw` format too.

Reply:
Do you want me to handle the file types explicitly in this PR or shall we do that later?

Reply:
lets do another PR, this PR is too big already